### PR TITLE
fix(SharePopup): use png encoding for avatars

### LINKED ISSUE #238: Some avatars do not show in the recap image
Checking the console, I've seen that it's mostly because of the avatars being in .webp or .gif format.
I could suggest you to change the extension at the end of the avatar URLs to fix it, as it's a Discord CDN's feature.

My recap image:
dumpus-share (2) (1)

### PR DESCRIPTION
**Pull request by skdhg**
closes #238 

image

### CODE REVIEW

File: src/components/StaticShareImage.tsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ export default function StaticShareImage({
           <div tw="flex mb-8">
             {/* eslint-disable-next-line */}
             <img
-              src={user.avatarURL}
+              src={user.avatarURL.replace(/.webp|.gif/, ".png")}
```
Comment (florian-lefebvre) [inline src/components/StaticShareImage.tsx:114]: Is this required? I think it's in the right format already because of your updates in SharePopup#L43-L46

Comment (skdhg) [reply src/components/StaticShareImage.tsx:114]: yeah I texted about this in [discord chat](https://discord.com/channels/558328638911545423/1127595753360076982/1131623858206679222). Maybe I was getting cached result, we can remove this.

Comment (florian-lefebvre) [reply src/components/StaticShareImage.tsx]: Yes I saw this and I'm pretty sure it's because of the cache so it's safe to remove it.

**Review by florian-lefebvre — APPROVED** [review]
LGTM

**Pull request merged**